int64 id
float64 x
float64 y
float64 yaw
string mode
---
bool success
string message
